packages/core/src/lib/banner/banner.svelte

```diff
@@ -110,10 +104,7 @@ $: {
         />
       {/if}
 
-      <figure
-        class="flex flex-col"
-        role="alert"
```

Comment:
From what I can tell from reading the specs, for `aria-live` regions like `alert` and `status`, screen readers announce dynamic changes to the elements, which means the `role=alert` element needs to be on the page first, _then_ the alert contents need to appear.

This, combined with the highly presentational nature of `Banner` lead me to make the following changes:

- I removed `role="alert"` from `Banner`
- I added `role="alert"` to `NotificationContainer`

This marks the `NotificationContainer` as a live element, and any time a new `Banner` (via `NotificationItem`) appears in the container, it should be announced as an alert. 

One remaining question: why is `Banner` a `<figure>` + `<figcaption>`? I haven't seen them used like this before for text

Reply:
> One remaining question: why is Banner a <figure> + <figcaption>? I haven't seen them used like this before for text

I actually meant to ask about this when I did the `Notify -> Banner` change and totally forgot. It looks like this is a carry over from `legacy` but I also am not sure why they are being used. Since they are not being used to caption any specific piece of media, feels like they could be replaced.